src/sql/SqlResult.ts

```diff
@@ -255,7 +257,11 @@ export class SqlResultImpl implements SqlResult {
         return this.closeDeferred.promise;
     }
 
-    /** Called when next page of the result is received. */
+    /**
+     * Called when next page of the result is received.
+     * @param page
+     * @internal
```

Comment:
Do we need to mark private functions as internal?

Reply:
also the class is internal anyway, going to delete it

Reply:
I am going to delete @internal labels of all methods in an internal class